.travis.yml

```diff
@@ -4,7 +4,6 @@ node_js:
 install:
   - npm install
 script:
-  - npm run package
```

Comment:
`npm install` runs prepublish. I can make it more explicit, but it'll run twice.